### PR TITLE
chore: use iframe to image on code-checker

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Related: https://github.com/kodadot/nft-gallery/issues/10524#issuecomment-2182335676

### CODE REVIEW

**Comment by hassnian** [comment]
something is wrong 

CleanShot 2024-06-22 at 10 25 52

**Comment by preschian** [comment]
> something is wrong , canary is working fine

oh, did you try on koda-art preview? for now try on netlify preview instead

the reason is, public/sandbox.html doesn't work on nuxt + cf-pages: https://github.com/kodadot/nft-gallery/blob/main/public/sandbox.html

on cf-pages, we proxy the routes to static bucket in this worker: https://github.com/kodadot/workers/blob/2fd6a8e1b75b724aab3c6c7ee11b1aabaebf5b67/services/ogi-route/src/index.ts#L7-L9

**Comment by preschian** [comment]
additional context, one artist need this

Screenshot 2024-06-25 at 12 24 32

File: components/codeChecker/massPreview/Capture.vue [hunk 1]
```diff
@@ -125,20 +120,16 @@ const initScreenshot = () => {
...
+  previewItems.value.forEach(async (preview) => {
     try {
       let url = getObjectUrl(indexKey.value!)
-
       url += `?hash=${preview.hash}`
 
-      preview = { ...preview, startedAt: performance.now() }
-
-      const response = await makeScreenshot(url)
+      const response = `https://iframe-to-blob.pages.dev/iframe?url=${url}`
```
Comment (hassnian) [inline components/codeChecker/massPreview/Capture.vue:128]: can we move this url into `capture.ts`

**Review by hassnian — APPROVED** [review]
otherwise LGTM

**Review by vikiival — COMMENTED** [review]
🧹 the constants and LGTM

File: components/codeChecker/massPreview/Capture.vue (hunk 1 above)
Comment (preschian) [reply components/codeChecker/massPreview/Capture.vue]: sure, updated

**Review by vikiival — APPROVED** [review]

**Pull request merged**